source/git/git_package_release_guide.rst

```diff
@@ -0,0 +1,71 @@
+2. `MINOR` refers to either new features, functional updates, or a large number of bundled bug fixes.
+3. `PATCH` refers to small bug fixes, hotfixes, or minor changes in the code. 
+4. Optional tag `<alpha/beta>` (angular braces `<>` indicate optional keywords) - This indicates the package has been released but is currently being tested and may not be stable. `alpha` testing usually implies testing within the broader community of the team that developed the package, whereas `beta` testing implies a wider release to the audience/end users, who can then report bugs which can be fixed. 
+
+On github, creating a release usually entails creating a stable version of the package on a branch, then creating a release tag for the specific stable commit. These branches can be standard main/develop or a dedicated release branch, which is usually dependent on the conventions of the organization creating the package. 
+
+When numbering versions, in rare cases, a package may need to be released before it is tested, or even when it is in active development. These should be tagged as a `v0.MINOR.PATCH<alpha>` release and it is good practice to highlight that the release is unstable or in active development. Once a package has a first stable release, the leading digit should roll over to `v1.MINOR.PATCH<alpha>`. For all subsequent releases, the first digit should only roll over once backward compatibility is broken, so subsequent tags should go as `v1.0.0`, `v1.1.0`, `v1.2.0`, etc... 
+
+Pre-release, release branches and archiving releases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Note that the tagging of `v0.MINOR.PATCH<alpha>` is not the same as creating a Pre-release on github, as tagging a commit is not the same as creating a release. UASAL organization preference is to *NOT create Pre-release versions* of packages.
+
+UASAL organization preference is to tag a release on the main branch. Prior to release, all feature branches should be merged into a develop branch, and then these changes should be pushed to the main branch and then tagged as a release from there. 
+
+When releasing code alongside a paper, a user can archive a copy of the github code on Zenodo. Zenodo usually generates two DOIs for each archived code, one for the code as a whole, and one for the specific release tag. So, creating a release tag will create a new snapshot of the archived code on Zenodo. Thus, in future papers, authors can choose to reference the package as a whole, or specific versions of code that were used for analysis, provided they were tagged as a github release to trigger the Zenodo archiving process.
+
+Checks before releasing a package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prior to release, consider the following checks on your github repo:
+
+1. Is the package on a stable release branch? Or will the package release be on the main/develop branch of the repo?
```

Comment:
I think 1 and 2 are no longer needed since we are recommending to release from main?